contracts/drops/token/HolographDropERC721.sol

```diff
@@ -755,12 +756,6 @@ contract HolographDropERC721 is NonReentrant, ERC721H, IHolographDropERC721 {
     emit OpenMintFinalized(msgSender(), config.editionSize);
   }
 
-  function setDropsPriceOracle(address newDropsPriceOracle) external onlyOwner {
```

Comment:
Are we cool with no longer being able to set this?

Reply:
I think it's mandatory to have it hardcoded. Since the drops contract is provided via proxy implementation, we can always upgrade the source with new address, if we need to change it for some reason.

Reply:
@ACC01ADE @alexanderattar if this is removed where is this being called? are we doing this part of the initializations?